### DISCUSSION TITLE
List inboxes available to a particular agent in their settings

### DISCUSSION THREAD

**Discussion by pranavrajs** [chatwoot/chatwoot#8194] [Feature Requests]
- Add a separate settings page for each agent.
- List the inboxes where the agent has access to.
- Add the UI to provide access inboxes from agent settings.
- Add API to list inboxes accessible to an agent
- Add API to add/remove inbox permission to an agent.

**Comment by murtaza-bagwala** [comment]
@pranavrajs shall I pick the last 2 tasks which are related to creating the APIs ?
